package.json

```diff
@@ -17,6 +18,7 @@
     "lint:css": "stylelint \"src/**/*.{svelte,css}\""
   },
   "dependencies": {
+    "@babel/runtime": "7.10.1",
```

Comment:
Why is this needed?

Reply:
It is a production dependency that replaces `@babel/plugin-transform-runtime`. https://babeljs.io/docs/en/babel-plugin-transform-runtime

Reply:
Oh I see now, you are updating the existing `babel-runtime` dependency.